### DISCUSSION TITLE
Reactivate Simplelogin after canceling Proton

### DISCUSSION THREAD

**Discussion by pehlm** [simple-login/app#1698] [Q&A] [answered]
Hi, I have Simplelogin via Proton. I plan to cancel Proton service but keep Simplelogin, how do I reactivate Simplelogin subscription after canceling Proton?

Regards

**Comment by nguyenkims** [comment]
@pehlm when you cancel in Proton, your past subscription will take over. When it ends, you can take another subscription.

**Reply by pehlm** [reply]
@nguyenkims, thanks for the answer. When I think about it I don't think I have any subscription with Paddle Simplelogin left now. Can I cancel Proton and then immediately subscript again to Simplelogin without lose anything?

**Reply by nguyenkims** [reply, marked as answer]
Yes, all your existing aliases will continue working normally in the meantime.
